hack/setup-cluster.sh

```diff
@@ -614,6 +620,12 @@ deploy() {
   echo "${bright}Done deploying manifests from current worktree on cluster ${CLUSTER_NAME}${reset}"
 }
 
+reload() {
```

Comment:
This is not strictly related to this fix, please let's keep the pr as simple as possible.

Reply:
Both this function and the watch-cluster.sh have been used to implements the e2e test, they are pretty related to the development; should I open a PR for this two file? They would be used for other contributions.

Reply:
It's usually better to keep PRs focused at first, to both help the maintainers reviewing and the author increasing the chances to see their contribution merged quickly. I'd suggest you to keep them for a future dedicated contribution.

Reply:
Removed then :)